python/llm/example/CPU/Deepspeed-AutoTP/README.md

```diff
@@ -57,7 +59,7 @@ If you want to run your own application, there are **necessary configurations in
 
 ```bash
 # run.sh
-source bigdl-nano-init
+source bigdl-llm-init
```

Comment:
Do we need to use "tcmalloc"?

Reply:
we tend to use default llm-init in the example